### PR TITLE
MTA-552 mta 6 2 0 release notes hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
Preview:
https://deploy-preview-713--windup-documentation.netlify.app/docs/release-notes/master/index.html

### CODE REVIEW

File: docs/topics/mta-rn-resolved-issues-6-2-0.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-6.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-6-2-0_{context}"]
+= Resolved issues
+
+The following issues have been resolved in MTA version 6.2.0.
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-6-2-0.adoc:9]: Looking at the format use in [existing documentation](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.1/html/release_notes/mta-6-1-0#mta-rn-resolved-issues-610_release-notes) please can you use this format for now? 

I honestly prefer the Jira filter than the list

File: docs/topics/mta-rn-resolved-issues-6-2-0.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+* link:https://issues.redhat.com/browse/MTA-721[MTA-721] - Create infra automation for Jira integrastion
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-6-2-0.adoc:11]:
```suggestion
* link:https://issues.redhat.com/browse/MTA-721[MTA-721] - Create infra automation for Jira integration
```

Comment (anarnold97) [reply docs/topics/mta-rn-resolved-issues-6-2-0.adoc:11]: I think you will find this is not a bug but a task.

File: docs/release-notes/master.adoc [hunk 3]
```diff
@@ -13,10 +13,10 @@ include::topics/templates/document-attributes.adoc[]
...
 include::topics/making-open-source-more-inclusive.adoc[]
 
-include::topics/mta-rn-new-features.adoc[leveloffset=+1]
+include::topics/mta-rn-new-features-6-2-0.adoc[leveloffset=+1]
 
-include::topics/mta-rn-known-issues.adoc[leveloffset=+1]
+include::topics/mta-rn-known-issues-6-2-0.adoc[leveloffset=+1]
 
-include::topics/mta-rn-resolved-issues.adoc[leveloffset=+1]
+include::topics/mta-rn-resolved-issues-6-2-0.adoc[leveloffset=+1]
```
Comment (anarnold97) [inline docs/release-notes/master.adoc:20]: You have put your release notes in the wrong place

File: docs/topics/mta-rn-new-features-6-2-0.adoc [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
+
+.Integration with JIRA
+A new _Jira_ entry has been added to the in the left sidebar, under the Control Menu. This feature enables the user (Administrator, Architect) to create new Jira instances and edit existing ones.
```
Comment (anarnold97) [inline docs/topics/mta-rn-new-features-6-2-0.adoc:12]:
```suggestion
A new _Jira_ entry has been added to the left sidebar, under the Control Menu. This feature enables the user (Administrator, Architect) to create new Jira instances and edit existing ones.
```

What do you mean by `edit`?

File: docs/release-notes/master.adoc [hunk 5]
```diff
@@ -13,11 +13,26 @@ include::topics/templates/document-attributes.adoc[]
...
 include::topics/making-open-source-more-inclusive.adoc[]
 
+:!release-notes:
+
 [id="release-notes-introduction"]
 == Introduction
 {ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
 
-These release notes cover all _z_-stream releases of {ProductShortName} 6.1 with the most recent release listed first.
+These release notes cover all _z_-stream releases of {ProductShortName} 6.1 and 6.2, with the most recent release listed first.
```
Comment (anarnold97) [inline docs/release-notes/master.adoc:22]: No it is release notes for 6.2. 
Think:
Release notes- 
- 6.2.x
- 6.1.x
- 6.0.x

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:9]: Please can you follow the existing format for MTA - https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.1/html/release_notes/mta-6-1-0#rn-known-issues-610_release-notes

image

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+* link:https://issues.redhat.com/browse/MTA-823[MTA-823] - Report custom metrics on MTA usage back via OpenShift Telemetry
+* link:https://issues.redhat.com/browse/MTA-402[MTA-402] - Report custom metrics on MTA usage back via OpenShift Telemetry
+* link:https://issues.redhat.com/browse/MTA-402[MTA-402] - Downstream efforts for MTA 6.2.0
+* link:https://issues.redhat.com/browse/MTA-894[MTA-894] - Failed analysis counted twice
+* link:https://issues.redhat.com/browse/MTA-872[MTA-872] - After an error, fails to export same applications as tastks, returns error showing sub-tasks
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:15]: Use Vale or check for typos
```suggestion
* link:https://issues.redhat.com/browse/MTA-872[MTA-872] - After an error, fails to export same applications as **tastks**, returns error showing sub-tasks
```

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://issues.redhat.com/browse/MTA-973[MTA-973] - Success alert missing when creating a new Jira instance
+* link:https://issues.redhat.com/browse/MTA-810[MTA-810] - Fails to run analysis in provate GitHub repo
+* link:https://issues.redhat.com/browse/MTA-984[MTA-984] - Dependencies: error retreiving data with `Unable to Connect`
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:18]: * link:https://issues.redhat.com/browse/MTA-984[MTA-984] - Dependencies: error **retrieving** data with `Unable to Connect`

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://issues.redhat.com/browse/MTA-696[MTA-696] - Exporting a migration wave to Jira causes all applications to enter Error state before creating issues
+* link:https://issues.redhat.com/browse/MTA-906[MTA-906] - Exporting Migration Wave as an Epic does not export to Jira - unsing Jira Server/DataCenter
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:20]: **Unsing** or **Using**

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://issues.redhat.com/browse/MTA-870[MTA-870] - A Migration Wave cannot be exported as a SubTask - using both Jira DataCenter and Cloud
+* link:https://issues.redhat.com/browse/MTA-465[MTA-465] - Tags and Reports tabs for application keeps loading while analysis in progress
+* link:https://issues.redhat.com/browse/MTA-1014[MTA-1014] - Cannot remove Jira connection afater it was added to Wave even after Wave was removed
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:23]: **afater** or **after**

File: docs/topics/mta-rn-product-recommendations-6-2-0.adoc [hunk 11]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
```
Comment (anarnold97) [inline docs/topics/mta-rn-product-recommendations-6-2-0.adoc:1]: Let us lose this page, if it is blank

File: docs/topics/mta-rn-technical-preview-6-2-0.adoc [hunk 12]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
```
Comment (anarnold97) [inline docs/topics/mta-rn-technical-preview-6-2-0.adoc:1]: If it does not have any technical previews (MTA rarely does) let us not include.

We need to say if there are:
- **New features and improvements**
- **Known Issues**
- **Resolved issues**

If there are **Technical Previews** or **Product Recommendations** we can include those pages but as there aren't it looks a bit weird.

File: docs/topics/mta-rn-new-features-6-2-0.adoc [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
+
+.Integration with JIRA
+A new _Jira_ entry has been added in the left sidebar, under the Control Menu. This feature enables the user (Administrator, Architect) to create new Jira instances and edit existing instances.
```
Comment (anarnold97) [inline docs/topics/mta-rn-new-features-6-2-0.adoc:12]: I do not think this is a good description of the feature. I have an issue, pedantic albeit, with lists in brackets as I do not think it is good grammar. 

@sbeskin-redhat - please could you help with the wording here?

Comment (sbeskin-redhat) [reply docs/topics/mta-rn-new-features-6-2-0.adoc:12]: Yes, I am reviewing these release notes.

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+* link:https://issues.redhat.com/browse/MTA-823[MTA-823] - Report custom metrics on MTA usage back via OpenShift Telemetry
```
Comment (sbeskin-redhat) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:11]: @HagayVider1 
This not a known issue. This is a new feature.
Do you have a list of known features from QE?

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://issues.redhat.com/browse/MTA-402[MTA-402] - Report custom metrics on MTA usage back via OpenShift Telemetry
```
Comment (sbeskin-redhat) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:12]: @HagayVider1 
Same as above. This not a known issue. This is a new feature.

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+* link:https://issues.redhat.com/browse/MTA-696[MTA-696] - Exporting a migration wave to Jira causes all applications to enter Error state before creating issues
+* link:https://issues.redhat.com/browse/MTA-906[MTA-906] - Exporting Migration Wave as an Epic does not export to Jira - unsing Jira Server/DataCenter
+* link:https://issues.redhat.com/browse/MTA-870[MTA-870] - A Migration Wave cannot be exported as a SubTask - using both Jira DataCenter and Cloud
+* link:https://issues.redhat.com/browse/MTA-465[MTA-465] - Tags and Reports tabs for application keeps loading while analysis in progress
+* link:https://issues.redhat.com/browse/MTA-1014[MTA-1014] - Cannot remove Jira connection afater it was added to Wave even after Wave was removed
+* link:https://issues.redhat.com/browse/MTA-1015[MTA-1015] - Credential filtering is missing _Created by_ filter
+* link:https://issues.redhat.com/browse/MTA-849[MTA-849] - Unable to connect to Jira instance behind a proxy
+
+See an updated  filtered list of link:https://issues.redhat.com/issues/?filter=12418204[known issues in Jira].
```
Comment (sbeskin-redhat) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:27]: @HagayVider1 
This link doesn't seem to work.
Also, you don't have to give a list of the issues if you provide a link to the filter.

File: docs/topics/mta-rn-new-features-6-2-0.adoc (hunk 13 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-rn-new-features-6-2-0.adoc:12]: @HagayVider1 
I would suggest this description of this feature:
"Integration with Jira enables users to track migrations of individual applications by creating a Jira issue for each migration from within the {ProductShortName} {WebName}."

File: docs/topics/mta-rn-new-features-6-2-0.adoc [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+
+.Report custom metrics on MTA usage back through OpenShift Telemetry
+MTA reports back through OpenShift Telemetry as to the applications that it processes:
+
+* Number of applications in Inventory
+* Number of Analysis Run (aggregated)
+* Number of Assessments Run
+
```
Comment (sbeskin-redhat) [inline docs/topics/mta-rn-new-features-6-2-0.adoc:20]: @HagayVider1 
The details of this feature do not belong in the MTA documentation. We should only mention the following as a new feature:
".OpenShift Monitoring integration
This release supports OpenShift Monitoring integration."

File: docs/topics/mta-rn-new-features-6-2-0.adoc [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+.Migration Waves management
+The technical lead of the migration project can create and modify applications and information on the migration.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-rn-new-features-6-2-0.adoc:22]: @HagayVider1 
I would suggest this:
".Migration waves
A migration wave is a tool for grouping applications to be migrated on a given schedule. In addition, a migration wave enables the user to export a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave for tracking."

File: docs/topics/mta-rn-known-issues-6-2-0.adoc (hunk 14 above)
Comment (anarnold97) [reply docs/topics/mta-rn-known-issues-6-2-0.adoc:11]: Not yet! But we want to be ahead of the curve

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 19]
```diff
@@ -0,0 +1,17 @@
...
+
+MTA version 6.2.0 has the following issues.
+
+.MTA fails to run analysis in private GitHub repo
+Fail to run analysis in private github repo. link:https://issues.redhat.com/browse/MTA-810[MTA-810]
+
+.Jira connection (instance) cannot be removed 
+Jira connections cannot be removed after it was added to Wave even after Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]  
+
+See an updated  filtered list of link:https://issues.redhat.com/issues/?filter=12418204[known issues in Jira].
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:17]: @HagayVider1 - here is how I would like the Known Issues to look. What do you think?

Please reach out to Dylan and ask him if there are any other known issues he wants adding? Also ask for more details about the two known issues (if he wants them keeping)

Please can you check the Jira filter and see if you can figure out what is going wrong as I cannot access it.

File: docs/topics/mta-rn-resolved-issues-6-2-0.adoc [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+
+.Analysis wizard
+The release of MTA 6.2.0 resolves the issue that Analysis wizard was stuck on custom rules page on moving *Back* from the Repository tab.
+For more information on this issue, see link:https://issues.redhat.com/browse/MTA-464[MTA-464].
+
+.Tags & Reports tabs
+The release of MTA 6.2.0 resolves the issue that an analysis was running for an application, and after clicking on that application to see the Tags and Reports, both the tabs keep loading until the analysis finished.
+For more information on this issue, see link:https://issues.redhat.com/browse/MTA-465[MTA-465].
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12418203[Resolved Issues in Jira].
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-6-2-0.adoc:19]: Thanks @HagayVider1 looks better. I think you need to tighten up the language and format it better.
But it works well as a draft.

File: docs/topics/mta-rn-new-features-6-2-0.adoc [hunk 21]
```diff
@@ -0,0 +1,18 @@
...
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
+
+.Integration with JIRA
+A new _Jira_ entry has been added in the left sidebar, under the Control Menu. This feature enables the user (Administrator, Architect) to create new Jira instances and edit existing instances.
```
Comment (abrugaro) [inline docs/topics/mta-rn-new-features-6-2-0.adoc:12]: I may be missing something here, but:

`under the Control Menu`

The Jira entity is under a new `Issue Management` menu in the Administration Perspective

**Review by abrugaro — CHANGES_REQUESTED** [review]

File: docs/topics/mta-rn-known-issues-6-2-0.adoc [hunk 22]
```diff
@@ -0,0 +1,18 @@
...
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.MTA fails to run analysis in private GitHub repo
+
+Fail to run analysis in private github repo. link:https://issues.redhat.com/browse/MTA-810[MTA-810]
+
+.Jira connection (instance) cannot be removed 
+Jira connections cannot be removed after it was added to Wave even after Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:16]: *Jira **connections...*** plural .... ***it was** added* is singular

Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-6-2-0.adoc:16]:
```suggestion
Jira connections cannot be removed after they are added to a _Migration Wave_, even after the Wave has been removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]  
```

**Pull request closed without merging**